administrator/language/en-GB/plg_system_webauthn.ini

```
@@ -41,5 +41,4 @@ PLG_SYSTEM_WEBAUTHN_MANAGE_HEADER_ACTIONS_LABEL="Actions"
 PLG_SYSTEM_WEBAUTHN_MANAGE_HEADER_NOMETHODS_LABEL="No authenticators have been set up yet."
 PLG_SYSTEM_WEBAUTHN_MSG_DELETED="The authenticator has been removed."
 PLG_SYSTEM_WEBAUTHN_MSG_SAVED_LABEL="The label has been saved."
-PLG_SYSTEM_WEBAUTHN_REQUIRES_GMP="The PHP Extension GMP is required to be loaded in order to add authenticators."
-
+PLG_SYSTEM_WEBAUTHN_REQUIRES_GMP="One of the PHP extensions GMP or BCmath must be loaded to add authenticators."
```

```suggestion
PLG_SYSTEM_WEBAUTHN_REQUIRES_GMP="Either of the PHP extensions GMP or BCmath must be loaded to add authenticators."
```

This is a super pedantic grammar change. Use "either" when there are only two options. Use "any of" when there are more than to

---

I thought about it but "either" sounds like an "exclusive or" to me, i.e. I need to have only the GMP extension installed or only the BCmatch extension involved. I used "One of" to signal that you can have only GMP, only BCmath or both.

Given this additional context do you think I should still make the change?

---

Maybe "At least one of the extensions GMP or BCmath ..."?

---

@nikosdion and to me it sounds the opposite. I stick by my grammar teachers